playbooks/byo/preflight/action_plugins/verify_openshift_image.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
```

Comment:
I'd leave these two lines out, AFAICT they have no effect here.

OTOH, upstream seems to have a :heart: relationship with vim: https://github.com/openshift/openshift-ansible/blob/master/docs/best_practices_guide.adoc#python-source-files, so we'll likely be asked to add those lines.